plugin/evm/vmsync/registry_test.go

```diff
@@ -294,5 +364,13 @@ func TestSyncerRegistry_NoSyncersRegistered(t *testing.T) {
 	ctx, cancel := utilstest.NewTestContext(t)
 	t.Cleanup(cancel)
 
-	require.NoError(t, registry.RunSyncerTasks(ctx, &client{}))
+	require.NoError(t, registry.RunSyncerTasks(ctx, getTestClientSummary(t)))
+}
+
+func getTestClientSummary(t *testing.T) message.Syncable {
```

Comment:
(nit, readability) How about `newDummyClientSummary`? Rationale:

The `get` prefix doesn't add anything informative while the `new` prefix suggests a constructor. In other scenarios "compute", "fetch", etc. are valuable to signal to the reader what the function is doing if more expensive or remote, respectively.

The `Test` element is also implied by the fact that we're in a test so could either be left out entirely or replaced with something informative like `Dummy` to indicate that it's a stand-in.